packages/pulp/python-pulpcore/python-pulpcore.spec

```diff
@@ -0,0 +1,82 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name pulpcore
+
+Name:           python-%{pypi_name}
+Version:        3.0.0rc8
+Release:        1%{?dist}
+Summary:        Pulp Django Application and Related Modules
+
+License:        GPLv2+
+URL:            http://www.pulpproject.org
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+
+%description
+[![Build Status]( [![PyPI]( [![codecov]( [![Code Quality: Python]( [![Total
```

Comment:
Given how central this package is, this probably deserves some special attention.